spring-aot/src/main/java/org/springframework/nativex/type/TypeSystem.java

```diff
@@ -311,7 +323,8 @@ public Type resolve(org.objectweb.asm.Type type, boolean silent) {
 	public Type Lresolve(String desc, boolean silent) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.